src/org/org.ts

```diff
@@ -590,11 +590,12 @@ export class Org extends AsyncOptionalCreatable<Org.Options> {
         this.options.aliasOrUsername = aliasOrUsername || undefined;
       }
 
-      const username = this.options.aliasOrUsername;
+      const aliases = globalInfo.getAliasOf(this.options.aliasOrUsername as string);
+      const username = Array.isArray(aliases) ? aliases[0] : aliases ?? this.options.aliasOrUsername;
```

Comment:
There seems to be a lot of array checking - should we add a method that will always return the first alias so that we don't have to keep checking if it's an array or not?

Reply:
Yeah, I wasn't a huge fan of this either. Reversing the aliases (e.g. `{ 
"my-user": "my-user-alias" }` made this a lot simpler but ended up backtracking on that path because of the collision thing I told you about.

I'll take another swipe at simplifying the return of `GlobalInfo.prototype.getAliasesOf()` after I do some QA and I can get the current unit tests passing.